CRM/Contact/BAO/Contact.php

```diff
@@ -775,16 +775,6 @@ protected static function contactTrash($contact): bool {
    *
    */
   public static function resolveDefaults(&$defaults, $reverse = FALSE) {
-    // Hack for birth_date.
```

Comment:
This function is only called from the import parser. We already added extensive cover for birth_date handling so this can go along with the options